sdk/eventgrid/eventgrid/review/eventgrid.api.md

```diff
@@ -99,47 +131,121 @@ export type ContainerRegistryImageDeletedEventData = ContainerRegistryEventData
 export type ContainerRegistryImagePushedEventData = ContainerRegistryEventData & {};
 
 // @public
-export class EventGridClient {
-    constructor(endpointUrl: string, credential: KeyCredential | SignatureCredential, options?: EventGridClientOptions);
-    readonly apiVersion: string;
-    readonly endpointUrl: string;
-    generateSharedAccessSignature(expiresOnUtc: Date): string;
-    sendCloudEvents(events: CloudEvent<any>[], options?: SendCloudEventsOptions): Promise<RestResponse>;
-    sendCustomSchemaEvents(events: Record<string, any>[], options?: SendCustomSchemaEventsOptions): Promise<RestResponse>;
-    sendEvents(events: EventGridEvent<any>[], options?: SendEventsOptions): Promise<RestResponse>;
+export type CustomEventDataDeserializer = (o: any) => Promise<any>;
+
+// @public
+export interface DeviceConnectionStateEventInfo {
+    sequenceNumber?: string;
+}
+
+// @public
+export interface DeviceConnectionStateEventProperties {
+    deviceConnectionStateEventInfo?: DeviceConnectionStateEventInfo;
+    deviceId?: string;
+    hubName?: string;
+    moduleId?: string;
+}
+
+// @public
+export interface DeviceLifeCycleEventProperties {
+    deviceId?: string;
+    hubName?: string;
+    twin?: DeviceTwinInfo;
+}
+
+// @public
+export interface DeviceTelemetryEventProperties {
+    body?: any;
```

Comment:
maybe `unknown` is better here?

Reply:
Added this to https://github.com/Azure/azure-sdk-for-js/issues/10992